src/Common/Dwarf.cpp

```diff
@@ -696,14 +696,13 @@ Dwarf::Die Dwarf::getDieAtOffset(const CompilationUnit & cu, uint64_t offset) co
     die.is64Bit = cu.is64Bit;
     auto code = readULEB(sp);
     die.code = code;
+    die.attr_offset = sp.data() - info_.data() - offset;
```

Comment:
The code might become incorrect.

What is "code"? Maybe it's impossible to reference `sp` and `cu` and `die` if the code is zero?